dateString.js

```diff
@@ -9,6 +9,39 @@ var MAX_YEAR = 2019
 function dateString(input, opts) {
   var id = hash(input)
 
+  opts = opts || 0
+  var minDate, maxDate
+
+  // keep older dateString function for backwards compatibility
+  if (!opts || (opts.minYear && opts.maxYear)) {
+    return dateStringLegacy(input, opts)
+  }
+
+  if (opts.min) {
+    minDate = new Date(opts.min)
+  } else {
+    var minYear = defaults(opts.minYear, MIN_YEAR)
```

Comment:
Assuming you meant [`||` above](https://github.com/oftherivier/fictional/pull/41/files#r1793943407), I'd guess we don't need to be checking `minYear` and `maxYear` anymore for the rest of the logic in this function, right?